crates/ethereum/primitives/src/transaction.rs

```diff
@@ -369,6 +384,12 @@ impl TransactionSigned {
     }
 }
 
+impl core::ops::DerefMut for TransactionSigned {
+    fn deref_mut(&mut self) -> &mut Transaction {
+        &mut self.transaction
+    }
+}
+
```

Comment:
use `derive_more::DerefMut` instead to derive this https://docs.rs/derive_more/latest/derive_more/